### PR TITLE
{geo}[foss/2023a] PALM v23.10

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3139.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/d74cc1946d8a60f27306efa096e0e2ea for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by VRehnberg** [comment]
Test report by @VRehnberg
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/VRehnberg/5a73e1b88182a6144f3e4fe81a9360e5 for a full test report.

**Review by laraPPr — APPROVED** [review]
lgtm

**Comment by laraPPr** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/p/PALM/PALM-23.10-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        'checksums': ['fe348d75f01c911f362b6d998c57b3d85b505876dde69db924f2c512c395c328'],
+        'modulename': 'mkdocs_macros',
+    }),
+    ('pymdown_extensions', '10.7.1', {
+        'checksums': ['c70e146bdd83c744ffc766b4671999796aba18842b268510a329f7f64700d584'],
+        'modulename': 'pymdownx.magiclink',
+    }),
+    ('termcolor', '2.4.0', {
+        'checksums': ['aab9e56047c8ac41ed798fa36d892a37aca6b3e9159f3e0c24bc64a9b3ac7b7a'],
+        # fix for: "Unknown classifier in field project.classifiers: Programming Language :: Python :: 3.13"
```
Comment (Flamefire) [inline easybuild/easyconfigs/p/PALM/PALM-23.10-foss-2023a.eb:83]: FYI: I found the missing requirement/(build)dependency: https://github.com/easybuilders/easybuild-easyconfigs/pull/25526

**Pull request merged**